packages/vuetify/src/iconsets/mdi-unocss.ts

```diff
@@ -0,0 +1,83 @@
+// @unocss-include
```

Comment:
I had exported a VPlay demo (to have a quickest setup) and applied those changes to verify if this file would be scanned by UnoCSS when it is burried in `node_modules`.

It seems to only scan the project files - which seems like a reasonable thing to do. Do you know how to force the scan? Does it require additional entry in the config file?

Reply:
Uhmm, we also need to configure the pipeline, `[jt]s` files excluded by default, it works only with vite/webpack:

https://unocss.dev/config/#pipeline

So, VPlay cannot use this icon set.